app/cli.py

```diff
@@ -155,6 +157,29 @@ def apikey():
         """apikey commands"""
         pass
 
+    @app.cli.command()
+    def check_bad_url():
+        """check for expired resource url"""
+        resources = Resource.query.all()
+        headers = {
+            'User-Agent': 'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) \
+                           AppleWebKit/537.36 (KHTML, like Gecko) \
+                           Chrome/79.0.3945.117 Mobile Safari/537.36'
+        }
+        processes = []
+
+        def print_bad_url(resource):
+            try:
+                res = requests.get(resource.url, headers=headers)
+                res.status_code > 400 and \
+                    print(f"resource_id: {resource.id} | resource_url: {resource.url}")
+            except Exception:
+                print(f"resource_id: {resource.id} | resource_url: {resource.url}")
+
+        with ThreadPoolExecutor(max_workers=20) as executor:
+            for resource in resources:
```

Comment:
processes = [r for r in resources if executor.submit(filter_expired, r)]

Reply:
fixed